### PR TITLE
fft-eval: update to the version with ath11k support

### PR DESCRIPTION
**Pull request by klukonin**
Latest fft-eval has some extra features such as ath11k support, double precision in json signal calculation, and cleanups (batctl references, minor bugfixes, etc )
This commit is a version and makefile update. Now the native project Makefile do the job (compilation and linking).

Maintainer: Nicolas Pace nicopace@altermundi.net 

Compile tested: Ubuntu 19.10, current OpenWrt master) 
Run tested: Unifi AP AC Mesh, current OpenWrt master, tests done) 

Description: Latest fft-eval has some extra features such as ath11k  support, double precision in json signal calculation,   and cleanups  (batctl references, minor bugfixes, etc ) This commit is a version and makefile update. Now the native project Makefile do the job (compilation and linking).

### CODE REVIEW

**Comment by klukonin** [comment]
Original message from @neheb:
" @aparcar where did this maintainer come from? "

https://github.com/openwrt/packages/pull/11059#issuecomment-576096120

**Comment by klukonin** [comment]
BTW,
I have some plans to improve this package and can take part in maintaining process.
Or I can become a maintainer.

**Comment by neheb** [comment]
Go for it.

**Comment by klukonin** [comment]
@nicopace

Is it OK?

**Comment by nicopace** [comment]
I never took it as maintainer, just did one contribution (the JSON format).
Thanks for adding it, and happy if someone wants to maintain it!

**Comment by klukonin** [comment]
@neheb 
Done!

**Comment by neheb** [comment]
Works for me. Merging.

File: utils/fft-eval/Makefile [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
 endef
 
 define Package/$(PKG_NAME)/description
 	Evaluates FFT samples from diferent wifi boards drivers
 endef
 
-TARGET_CFLAGS  += -ffunction-sections -fdata-sections -flto
+TARGET_CFLAGS  += -Wall -W -std=gnu99 -fno-strict-aliasing -MD -MP
+TARGET_CPPFLAGS += -D_DEFAULT_SOURCE
+
```
Comment (BKPepe) [inline utils/fft-eval/Makefile]: Why empty row?

File: utils/fft-eval/Makefile [hunk 2]
```diff
@@ -1,39 +1,41 @@
...
+PKG_SOURCE_VERSION:=4d3b6faee428e3bd9f44ab6a3d70585ec50484a1
 PKG_SOURCE:=FFT_eval-$(PKG_SOURCE_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/simonwunderlich/FFT_eval/tar.gz/$(PKG_SOURCE_VERSION)?
-PKG_HASH:=9d70125fffc83918e6613d5938c4a9925aea07323e09470cf059cf0026e5937c
+PKG_HASH:=fb34ed5691af6c3ec09c41a6689ae64a1a12d770347a4effd48e4b30aef1eb22
 PKG_BUILD_DIR:=$(BUILD_DIR)/FFT_eval-$(PKG_SOURCE_VERSION)
 
 include $(INCLUDE_DIR)/package.mk
 
 define Package/$(PKG_NAME)
```
Comment (BKPepe) [inline utils/fft-eval/Makefile]: Don't re-use PKG_NAME.
More details: https://github.com/openwrt/packages/pull/4119

**Comment by klukonin** [comment]
Ok, I will improve it.

**Comment by aparcar** [comment]
> Hey @aparcar, how is that even possible? Ping also @dangowrt who merged your PR! #8265

This is possible via this commit https://github.com/libremesh/lime-packages/commit/f29428fbd88fc401899d4f8959d66de84eee2dcf in an effort to remove all C packages from the LibreMesh package feed. It mostly uses Lua which is architecture independent, so to simply the CI all arch specific packages where removed.

I copied the Makefile, thereby also @nicopace as a maintainer. I wasn't aware that he wasn't aware as this was a lengthy discussion some time ago over at LibreMesh. Sorry for the confusion @everyone involved. Glad a new maintainer took over!

**Pull request merged**